source/telemetry/release-notes/release-notes.md

```diff
@@ -1,5 +1,9 @@
 # Telemetry Release Notes
 
+## Version 2.4.0
```

Comment:
Why not 2.2.0?

Reply:
It would be nice if you fix the missing notes.

Reply:
There hasn't been other releases of telemetry, the other releases are for the sql migrations. So there are no missing notes.

The package shares release version across packages, which should be fixed, but is another task for another time in my opinion.

Reply:
I think it is because @BjarkeMeier see the release-notes in telemetry and sql migrations as separate versions from what is written as release tag.  I like your approach and think it gives more clarity on which version is being used of telemetry and / or sql migration, because you are able to link a release tag directly to a version of for example telemetry.